XiEditorTests/Mocks/DictionaryStorage.swift

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+class DictionaryStorage: KeyValueStorage {
+
+    var dictionary: [String : Any]
```

Comment:
why not private?

Reply:
@mmatoszko `DictionaryStorage` should be used for tests when you need to mock object that conforms to `KeyValueStorage` protocol. It is added to Tests target only.